posts/FLT-announcement.md

```diff
@@ -0,0 +1,81 @@
+---
+author: 'Kevin Buzzard'
+category: 'project announcement'
+date: 2024-04-30 18:00:00+00:00
+description: ''
+has_math: false
+link: ''
+slug: FLT-announcement
+tags: ''
+title: The Fermat's Last Theorem Project
+type: text
+---
+
+Kevin Buzzard discusses the project to prove Fermat's Last Theorem in Lean.
+
+<!-- TEASER_END -->
+# Introduction
+
+Fermat's Last Theorem (FLT) is the claim that some abstract equation has no solutions in positive integers.
+The result has no practical applications.
+So why did the announcement of its resolution by Andrew Wiles in 1993 make the front page of the New York Times?
+
+One aspect of FLT is that the theorem is very simple to *state* (`x^n+y^n=z^n` has no solutions in positive integers if `n>=3`), and yet it was incredibly hard to *prove* (indeed, it took the mathematical community over 350 years).
```

Comment:
nit: I think you can use `$` math here, unless the use of lean-like code is deliberate!

Reply:
I prefer the current version.

Reply:
I also don't know how to preview, but I think that's fine; it can be fixed in post if necessary.

Reply:
OK I've put it in maths mode and changed the false to true, we'll see what happens.